### PR TITLE
IBP-4676 Open Germplasm Details Page On Graph Node Click

### PR DESCRIPTION
**Pull request by abatac**
IBP-4676

### CODE REVIEW

File: src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm-details.url.service.ts [hunk 1]
```diff
@@ -25,4 +25,17 @@ export class GermplasmDetailsUrlService {
         return this.sanitizer.bypassSecurityTrustResourceUrl(GERMPLASM_DETAILS_URL + gid + queryParams);
 
     }
+
+    getUrlAsString(gid: any): string {
+
+        const queryParams = '?cropName=' + this.paramContext.cropName
```
Comment (darla-leafnode) [inline src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm-details.url.service.ts:31]: Can we extract this as constant:
`?cropName=' + this.paramContext.cropName            + '&amp;programUUID=' + this.paramContext.programUUID            + '&amp;authToken=' + this.paramContext.authToken            + '&amp;loggedInUserId=' + this.paramContext.loggedInUserId            + '&amp;selectedProjectId=' + this.paramContext.selectedProjectId;`

File: src/main/jhipster/src/main/webapp/app/shared/pedigree-graph/pedigree-graph.component.ts [hunk 2]
```diff
@@ -42,39 +47,79 @@ export class PedigreeGraphComponent implements OnInit {
...
+            if (!this.isUnknown(datum)) {
+                const node = group[i];
+                const selection = d3.select(node);
+                selection.selectAll('polygon').attr('fill', 'none');
+                selection.selectAll('text').attr('fill', '#000000');
+            }
+        });
+    }
+
+    isUnknown(datum): boolean {
```
Comment (darla-leafnode) [inline src/main/jhipster/src/main/webapp/app/shared/pedigree-graph/pedigree-graph.component.ts:115]: Rename as: `isUnknownGermplasm`

File: src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm-details.url.service.ts (hunk 1 above)
Comment (abatac) [reply src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm-details.url.service.ts]: done

File: src/main/jhipster/src/main/webapp/app/shared/pedigree-graph/pedigree-graph.component.ts (hunk 2 above)
Comment (abatac) [reply src/main/jhipster/src/main/webapp/app/shared/pedigree-graph/pedigree-graph.component.ts]: Done

**Review by darla-leafnode — APPROVED** [review]

**Pull request merged**